python/python/glide/async_commands/server_modules/ft.py

```diff
@@ -181,3 +181,52 @@ async def aliasupdate(
     """
     args: List[TEncodable] = [CommandNames.FT_ALIASUPDATE, alias, indexName]
     return cast(TOK, await client.custom_command(args))
+
+
+async def info(client: TGlideClient, indexName: TEncodable) -> Mapping[
+    TEncodable,
+    Union[
+        TEncodable,
+        int,
+        List[TEncodable],
+        List[
+            Mapping[
+                TEncodable,
+                Union[TEncodable, Mapping[TEncodable, Union[TEncodable, int]]],
+            ]
+        ],
+    ],
+]:
+    """
+    Returns information about a given index.
+
+    Args:
```

Comment:
Documentation matches JAVA. In this case, JAVA documentation is missing client in the params and needs to be updated.